packages/core/src/codewhisperer/util/codewhispererSettings.ts

```diff
@@ -64,6 +65,18 @@ export class CodeWhispererSettings extends fromExtensionManifest('amazonQ', desc
         return Math.max(this.get('workspaceIndexMaxSize', 250), 1)
     }
 
+    public getAutoBuildFeatureProjects(): { [key: string]: boolean } {
```

Comment:
It feels weird that now we are moving changes into the codewhisperer project to support this. Maybe we should seperate this into amazon q chat settings and amazon q inline settings? And then q inline can live here and amazon q chat can live in the amazon q folder